server/src-lib/Hasura/SQL/Types.hs

```diff
@@ -195,30 +195,34 @@ instance ToTxt TableName where
 data TableType
   = TTBaseTable
   | TTView
+  | TTMaterializedView
   | TTForeignTable
   | TTLocalTemporary
   deriving (Eq)
 
 tableTyToTxt :: TableType -> T.Text
-tableTyToTxt TTBaseTable      = "BASE TABLE"
-tableTyToTxt TTView           = "VIEW"
-tableTyToTxt TTForeignTable   = "FOREIGN TABLE"
-tableTyToTxt TTLocalTemporary = "LOCAL TEMPORARY"
+tableTyToTxt TTBaseTable        = "BASE TABLE"
+tableTyToTxt TTView             = "VIEW"
+tableTyToTxt TTMaterializedView = "MATERIALIZED VIEW"
+tableTyToTxt TTForeignTable     = "FOREIGN TABLE"
+tableTyToTxt TTLocalTemporary   = "LOCAL TEMPORARY"
```

Comment:
Generally good to try to avoid fixing alignment in a PR so that it's obvious what the semantic changes are. No big deal, but just harder to skim the changes.

Reply:
noted!